fuzz/src/fuzz_http.rs

```diff
@@ -0,0 +1,28 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use libfuzzer_sys::arbitrary::Arbitrary;
+use http::Request;
+use http::StatusCode;
+use http::Response;
+
+#[derive(Debug, Arbitrary)]
+struct HttpSpec {
+        uri: Vec<u8>,
+        header_name: Vec<u8>,
+        header_value: Vec<u8>,
+        status_codes: Vec<u8>,
+        response: Vec<u8>,
```

Comment:
Looks like this is just setting a `Vec` as the response body, which allows any type and any value. So I think generating this vector won't be useful for the fuzzing.

Reply:
Thank you! This should be solved now.